src/languages/en.ts

```diff
@@ -3204,9 +3205,12 @@ export default {
                             return 'Marking NetSuite bills and invoices as paid';
                         case 'intacctCheckConnection':
                             return 'Checking Sage Intacct connection';
+                        case 'intacctImportDimensions':
```

Comment:
This looks like it's missing a corresponding change in `es.ts`

Reply:
I checked es.ts and this key already existed

Reply:
Yeah, I noticed it when connecting to Sage Intacct. It showed "Translation missing for stage: intacctImportDimensions", so decided to fix it before somebody else notices this bug.

Reply:
Nice. Thanks for fixing that!